src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -966,7 +966,13 @@ protected function generateChanges() {
       throw new Exception(pht('Repository API is not supported.'));
     }
 
-    if (count($changes) > 250) {
+    $config = $this->getConfigurationManager();
+    $max_changes = $config->getConfigFromAnySource('differential.max_changes');
```

Comment:
for all uber specific configurations, @Pawka suggested me to use a "uber." prefix